src/vmware/HISTORY.md

```diff
@@ -1,5 +1,20 @@
 # Release History
 
+## 5.0.0 (2022-10)
+
+- Update to AVS 2022-05-01 API
+- Add `az vmware cluster list-zones`
+- Add `--affinity-strength` and `--azure-hybrid-benefit-type` parameters to `az vmware placement-policy vm-host create`
+- Add `--affinity-strength` and `--azure-hybrid-benefit-type` parameters to `az vmware placement-policy vm-host update`
+- Add `--express-route-id` parameter to `az vmware authorization create`
+- Add `vmware addon arc` command group
+- Add `az vmware location check-trial-availability`
+- Add `az vmware location check-quota-availability`
+- Deprecate `az vmware location checktrialavailability`
+- Deprecate `az vmware location checkquotaavailability`
+- [BREAKING CHANGE] `az vmware private-cloud add-availability-zone` has been removed. Availability zone cannot be modified after private cloud creation
+- [BREAKING CHANGE] `az vmware private-cloud delete-availability-zone` has been removed. Availability zone cannot be modified after private cloud creation.
```

Comment:
In fact, in order to avoid the impact of breaking change on the user's business, we usually suggest implementing the breaking change uniformly in the breaking change window, Azure CLI has bi-annual breaking change releases coinciding with Microsoft **Build** and **Ignite**. For more information about breaking change process, please refer to document [breaking_changes.md](https://msazure.visualstudio.com/DefaultCollection/One/_git/AzureCLITools-Wiki?path=/src/teams_docs/azcli_docs/breaking_changes.md&version=GBmain&_a=preview)

Reply:
Therefore, I suggest that you use `deprecate_info=g.deprecate(hide=True)` to hide these commands first, and then remove them in the next breaking change window *(next year's Build sprint, probably in May)*. This will give users more time to prepare

Reply:
Add availability zone does not work - availablility zone can only be modified during private cloud creation. These two commands don't do anything, any one customers that consume these aren't actually making any modifications.

Reply:
Should they still be deprecated?

Reply:
If you think this command is not used in the customer's scripts, I think it can be deleted directly. 
Otherwise, it is recommended to deprecate it first to avoid breaking the user's automatic scripts~

Reply:
They are not used. They can be removed.